### DISCUSSION TITLE
Is it possible to use component style layouts

### DISCUSSION THREAD

**Discussion by ilyosjon09** [tighten/jigsaw#576] [Q&A] [answered]
I tired to make a layout using component syntax but Jigsaw cannot locate the layout file. 

I tried the following snippet but it did not work:

`<x-main-layout>
    <h1>Hello world</h1>
 </x-main-layout>`

**Comment by bakerkretzmar** [comment, marked as answer]
Where did you put `main-layout.blade.php`? I think that should work if it's in `content/_components`.

**Reply by ilyosjon09** [reply]
It worked. Thank you much!
